smsdk/client.py

```diff
@@ -430,13 +478,25 @@ def get_cookbooks(self, **kwargs):
         )
         return cookbook(self.session, base_url).get_cookbooks(**kwargs)
 
-    def get_cookbook_top_results(self, recipe_group_id, limit=10, **kwargs):
+    def get_cookbook_top_results(self, *args, **kwargs):
```

Comment:
=None args

Reply:
This is the standard way of defining positional arguments. If there are no positional arguments passed, it will be come None. No need to set it explicitly.